src/pages/Index.tsx

```
@@ -1,54 +1,27 @@
-import { BookOpen, Users, Map, Calendar, Star, Headphones, LogIn, UserPlus, User } from "lucide-react";
-import { Link, useNavigate } from "react-router-dom";
-import SignInLink from '@/components/SignInLink';
-import { useAuth } from "@/contexts/authHelpers";
-import { useProfile } from "@/hooks/useProfile";
-import SEO from "@/components/SEO";
-import AnimatedHero from "@/components/AnimatedHero";
-import SahadhyayiCircuit from "@/components/SahadhyayiCircuit";
-import SahadhyayiCapabilities from "@/components/SahadhyayiCapabilities";
-import CurrentReads from "@/components/library/CurrentReads";
+import { Button } from '@/components/ui/button';
+import { BookOpen, Users, Map, Calendar, Star, Headphones } from 'lucide-react';
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)